src/index.js

```diff
@@ -110,7 +109,7 @@ if (APP_ROOT_NODE) {
                     <Route path={Routes.SURVEY} component={SurveyContainer} />
                     <Route path={Routes.QUESTIONNAIRE} component={QuestionnaireContainer} />
                     <Route path={Routes.TUD} component={TimeUseDiaryContainer} />
-                    <AuthRoute redirectToLogin path={Routes.ROOT} component={AppContainer} />
```

Comment:
Why does this need to be removed?

Reply:
I'm too lazy to transfer over the login app to the new aws stack. I might do it later on if it's necessary.